deploy/createTypesPackages.mjs

```diff
@@ -13,6 +13,42 @@ export const packages = [
         { from: "../generated/dom.iterable.generated.d.ts", to: "index.iterable.d.ts" }
       ],
     },
+    {
+      name: "@types/webworker",
```

Comment:
Probably need to resolve https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/997#discussion_r602899470 first before publishing webworker type, since currently it's a mixture of all types of workers:

https://github.com/microsoft/TypeScript-DOM-lib-generator/blob/bc78f9cffbc8cb9bcfce12a1780a77e9e2620344/src/build.ts#L253-L257

Reply:
OK, cool, I'll remove this one from the object right now which should unblock the PR